pkg/api/vlabs/validate.go

```diff
@@ -663,25 +660,13 @@ func (a *Properties) Validate(isUpdate bool) error {
 			case Swarm:
 			case SwarmMode:
 			case Kubernetes:
-				var version string
-				if a.HasWindows() {
```

Comment:
No need to check if the cluster has a Windows pool since this is run on Windows agents